corehq/ex-submodules/casexml/apps/case/const.py

```diff
@@ -45,3 +45,4 @@
 UNOWNED_EXTENSION_OWNER_ID = '-'
 
 ARCHIVED_CASE_OWNER_ID = '_archive_'
+INVALID_CASE_OWNER_ID = '_invalid_'
```

Comment:
Should also add this here: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/users/util.py#L21-L28